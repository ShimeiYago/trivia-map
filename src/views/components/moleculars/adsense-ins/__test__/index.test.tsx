import { shallow, ShallowWrapper } from 'enzyme';
import { AdsenseIns } from '..';

let wrapper: ShallowWrapper;

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(<AdsenseIns adSlot="xxx" adFormat="auto" />);
  });

  it('basic', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('with optional props', () => {
    wrapper.setProps({
      adLayoutKey: 'xxx',
      fullWidthResponsive: true,
    });
    expect(wrapper).toMatchSnapshot();
  });
});
