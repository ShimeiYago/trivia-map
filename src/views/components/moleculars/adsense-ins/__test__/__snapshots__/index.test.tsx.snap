// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests basic 1`] = `
<ins
  className="adsbygoogle"
  data-ad-format="auto"
  data-ad-slot="xxx"
  style={
    Object {
      "display": "block",
      "textAlign": undefined,
    }
  }
/>
`;

exports[`Shallow Snapshot Tests with optional props 1`] = `
<ins
  className="adsbygoogle"
  data-ad-format="auto"
  data-ad-slot="xxx"
  data-full-width-responsive="true"
  style={
    Object {
      "display": "block",
      "textAlign": undefined,
    }
  }
/>
`;
